### PR TITLE
ISSD-3113 [CP 2.0] Update product names in UI text and conditional checks 

### PR DESCRIPTION
**Pull request by pedroliveirav**
[ISSD-3113](https://liferay.atlassian.net/browse/ISSD-3113)

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json [hunk 1]
```diff
@@ -31,16 +31,16 @@
 	"all-projects": "All projects",
 	"allowed-email-domains": "Allowed Email Domains",
 	"almost-there-setup-analytics-cloud-by-finishing-the-activation-form": "Almost there! Setup Analytics Cloud by finishing the activation form.",
-	"almost-there-setup-experience-cloud-by-finishing-the-activation-form": "Almost there! Setup Experience Cloud by finishing the activation form.",
-	"almost-there-setup-lxc-sm": "Almost there! Setup LXC - SM",
+	"almost-there-setup-experience-cloud-by-finishing-the-activation-form": "Almost there! Setup Liferay SaaS by finishing the activation form.",
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json:34]: I don't think Brian is going to allow the lang key being different from the lang value (eg `experience-cloud=Liferay Saas`). While variable names can stay the same, lang keys won't make it past Brian's review. Can you please update all the lang key values as well

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+export function getProperProductsNames(name: string) {
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts:6]: rename to `getProperProductNames`. when variables are plural, usually the "s" only comes at the end, not in any of the words in the middle.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	if (/dxp/i.test(name)) {
+		return name.replace(/dxp/i, 'Liferay Self-Hosted');
+	}
+
+	if (/lxc - sm/i.test(name)) {
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts:11]: sort these if statements in alphabetical order

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/TableKeyDetails/index.jsx [hunk 4]
```diff
@@ -47,7 +48,7 @@ const TableKeyDetails = ({currentActivationKey, setValueToCopyToClipboard}) => {
 		currentActivationKey?.expirationDate
 	);
 
-	const formatedProductName = getFormatedProductName(
+	const formattedProductName = getFormatedProductName(
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/TableKeyDetails/index.jsx:51]: can you change the function name to be spelled correctly as well. `getFormattedProductName`

**Review by kylembischof — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json (hunk 1 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json]: Okay, done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts (hunk 2 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts (hunk 3 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getProperProductsNames.ts]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/TableKeyDetails/index.jsx (hunk 4 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/TableKeyDetails/index.jsx]: Done.

**Comment by pedroliveirav** [comment]
Please, could someone check the flows to see if anything is missing? I'd appreciate it. 

@kylembischof @samuelleolopes

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json [hunk 5]
```diff
@@ -420,8 +421,8 @@
 	"session-expired": "Session Expired",
 	"set-to-active": "Set to Active",
 	"set-up-analytics-cloud": "Set up Analytics Cloud",
-	"set-up-liferay-experience-cloud": "Set Up Liferay Experience Cloud",
-	"set-up-lxc-sm": "Set up LXC - SM",
+	"set-up-liferay-paas": "Set up Liferay PaaS",
+	"set-up-liferay-saas": "Set Up Liferay SaaS",
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json:425]: lowercase "Up" to match the key above it

**Comment by kylembischof** [comment]
I searched the code base and did not see any remaining "LXC" lang-key references. Nice job Pedro

**Comment by kylembischof** [comment]
@pedroliveirav you mentioned in daily meeting today that you were possibly going to send your banner ticket along side this one. Should we close this and you send a follow up PR with both tickets together?

**Comment by pedroliveirav** [comment]
resent: https://github.com/liferay-customer-delivery/liferay-portal/pull/62

**Pull request closed without merging**